DAO.sol

```diff
@@ -500,8 +500,19 @@ contract DAO is DAOInterface, Token, TokenSale {
 
         uint quorum = p.yea + p.nay;
 
+        // require 53% for callin newContract()
+        bool updateContractQuroumCheck = true;
```

Comment:
Update variable name in all occurences. `updateContractQuroumCheck` -> `updateContractQuorumCheck`